packages/fast-components-react-base/src/context-menu/context-menu.props.ts

```diff
@@ -0,0 +1,18 @@
+import * as React from "react";
+import {
+    ContextMenuClassNameContract,
+    ManagedClasses,
+} from "@microsoft/fast-components-class-name-contracts-base";
+import { ContextMenuItemProps } from "../context-menu-item";
+
+export interface ContextMenuManagedClasses
+    extends ManagedClasses<ContextMenuClassNameContract> {}
+export interface ContextMenuUnhandledProps extends React.HTMLAttributes<HTMLDivElement> {}
+export interface ContextMenuHandledProps extends ContextMenuManagedClasses {
+    /**
+     * The context-menu-item children

```

Comment:
I know these are just comments, but can we remove the hyphenation here and elsewhere?